plugin/spec-runner.vim

```diff
@@ -6,10 +6,35 @@ let s:COMMAND_FAILED = -1
 let s:NOT_IN_SPEC_FILE = -2
 let s:WARNING_UNABLE_TO_DETERMINE_RUNNER = 'Unable to determine correct spec runner'
 
+let g:spec_runner_available_runners = {
+      \ 'rspec' : 'rspec',
+      \ 'teaspoon' : { 'no_preloader': 'teaspoon', 'zeus': 'rake teaspoon' },
+      \ }
+
 if ! exists('g:spec_runner_dispatcher')
   let g:spec_runner_dispatcher = '!echo "{command}" && {command}'
 endif
 
+function! g:SpecRunner_detect_teaspoon()
```

Comment:
The tests went even weird than usual when I used autoloading and `SpecRunner#detect#teaspoon`, so this seemed easiest.